### PR TITLE
Pull #2: add two sections 

### PR DESCRIPTION
**Pull request by Ismailco**
add about me section and work section.
used flexbox  and grid to align elements on the page
update heading section (add border-radius  to bottom left)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,5 +51,171 @@ <h1 class="headline-header">I'm Ismail Glade to see you!</h1>
...
+          sign-ups required.
+        </p>
+        <div>
+          <span class="lang-section">html</span>
+          <span class="lang-section">css</span>
+          <span class="lang-section">javascript</span>
+          <br />
+          <button type="button" class="work-btn">See Project</button>
+        </div>
+      </div>
```
Comment (Temmarie) [inline index.html]: - [x] The list of projects should use a `ul` tag to group them, and `li` per project. Kindly use the appropriate semantic HTML tag for this as it aids with the code clarity.

File: index.html [hunk 2]
```diff
@@ -50,5 +51,171 @@ <h1 class="headline-header">I'm Ismail Glade to see you!</h1>
...
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div>
+          <span class="lang-section">html</span>
+          <span class="lang-section">css</span>
+          <span class="lang-section">javascript</span>
+          <br />
+          <button type="button" class="work-btn">See Project</button>
+        </div>
```
Comment (Temmarie) [inline index.html]: - [x] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") being a list should be ordered using `ul` and each of them a `li`.

File: index.html [hunk 3]
```diff
@@ -50,5 +51,171 @@ <h1 class="headline-header">I'm Ismail Glade to see you!</h1>
...
+    <section class="about-me-section">
+      <div class="about-me-content">
+        <h2 class="about-me-header">About Myself</h2>
+        <p class="about-me-description">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+          <br />
+          <a class="connect-link" href="#">Let's connect</a>
```
Comment (Temmarie) [inline index.html]: - [x] This is a title and should use a `hx` tag. You can wrap the `hx` tag around the `a` tag as it has to be a link as well.

File: index.html [hunk 4]
```diff
@@ -50,5 +51,171 @@ <h1 class="headline-header">I'm Ismail Glade to see you!</h1>
...
+          </div>
+          <br />
+          <hr />
+          <br />
+          <div class="collapsible-menu">
+            <button class="more-btn" type="button">Skills</button>
+          </div>
+          <br />
+          <hr />
+        </div>
```
Comment (Temmarie) [inline index.html]: - [x] The technology categories ("Languages", "Frameworks" and "Skills") are a list and should use  a `ul`, each of the category should be in a `li`
- [x] The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`. You are currently missing this section of the page, kindly refer to the template and then add it.
image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @Ismailco  ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Required Changes** ♻️

_See the inline comments below for more required changes_

### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.

-  N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

File: index.html [hunk 5]
```diff
@@ -50,5 +51,197 @@ <h1 class="headline-header">I'm Ismail Glade to see you!</h1>
...
+          <i class="info-description info-sep fa fa-circle"></i>
+          <p class="info-description">2015</p>
+        </div>
+        <p class="work-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="technologies">
+          <li class="lang-section">html</li>
+          <li class="lang-section">css</li>
```
Comment (MahmoudBakr23) [inline index.html]: - [ ] As the previous reviewer requested. Please make sure that each project (the whole project part) is defined in HTML inside a `<li>` tag and the `<li>` tags are wrapped inside a `<ul>` as a container list for all the projects' section. Kindly work on that 😄

File: index.html [hunk 6]
```diff
@@ -50,5 +51,197 @@ <h1 class="headline-header">I'm Ismail Glade to see you!</h1>
...
+
+                <li class="coll-item">
+                  <img
+                    class="coll-icon"
+                    src="./img/Icons/js.svg"
+                    alt="javascript-icon"
+                  />
+                  <p class="coll-text">JavaScript</p>
+                </li>
+                <li class="coll-item">
```
Comment (MahmoudBakr23) [inline index.html]: Good job adding the missing section but it still needs a little bit of improvement. Let's go through this section step-by-step.

1. 

- [x] The section (parent element) `<div class="more-info">` is supposed to be a `<section>` tag like you did in the other sections above it.

2. 

A list of Languages, Frameworks, and Skills. Each of them is wrapped inside a `<li>` within that list `<ul>`. this is correct right now 💯 👍 

3. 

- [x] The Languages element is supposed to be a header for another inner (smaller) list `<ul>`. Currently, you're adding the Languages header as an element in this smaller list. Kindly work on that issue.

4.

- [x] Kindly apply the same styling of the "Languages" header as the other elements in the same list and as the given design as well.

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @Ismailco 😃, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -50,5 +51,199 @@ <h1 class="headline-header">I'm Ismail Glade to see you!</h1>
         </ul>
       </div>
     </section>
+
+    <!-- the Works section -->
+
+    <section class="works-section">
+      <div class="work-card work-cardFirst">
```
Comment (smcommits) [inline index.html]: - [x] **Please use `<ul>` to group the project and `li` element to display each project**

What this means is that you should use `ul` to group all the project together in one element. Then, inside of that `ul` element, you should use `li` to display each project and it's contents. 

Why is it important? 

Whenever you are trying to group and list things, you should use `li` tags. HTML has the ability to express lists of things, and it helps the Google robot, screen readers, and all manner of users who don't care solely about the presentation of the site understand your content better.

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @Ismailco , 

### Great work on making the changes requested by a previous reviewer 👏 
### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 7 above)
Comment (Davidosky007) [reply index.html]: This has been solved 👍

**Comment by Davidosky007** [comment]
Hi @Ismailco, 

# Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**